Ginger/GingerCore/ALM/Qtest/ExportToQtest.cs

```diff
@@ -0,0 +1,640 @@
+                                        Activity matchingActivity = activities.Where(x => x.ExternalID == stepDesignID.ToString()).FirstOrDefault();
+                                        if (matchingActivity != null)
+                                        {
+                                            switch(matchingActivity.Status)
+                                            {
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed:
+                                                    step.Status = "Failed";
+                                                    List<IAct> failedActs= matchingActivity.Acts.Where(x => x.Status == Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed).ToList();
+                                                    string errors = string.Empty;
+                                                    foreach (Act act in failedActs)
+                                                    {
+                                                        errors += act.Error + Environment.NewLine;
+                                                    }
+                                                    step["ST_ACTUAL"] = errors;
+                                                    break;
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.NA:
+                                                    step.Status = "N/A";
+                                                    step["ST_ACTUAL"] = "NA";
+                                                    break;
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Passed:
+                                                    step.Status = "Passed";
+                                                    step["ST_ACTUAL"] = "Passed as expected";
+                                                    break;
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Skipped:
+                                                    step.Status = "N/A";
+                                                    step["ST_ACTUAL"] = "Skipped";
+                                                    break;
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Pending:
+                                                    step.Status = "No Run";
+                                                    step["ST_ACTUAL"] = "Was not executed";
+                                                    break;
+                                                case Amdocs.Ginger.CoreNET.Execution.eRunStatus.Running:
+                                                    step.Status = "Not Completed";
+                                                    step["ST_ACTUAL"] = "Not Completed";
+                                                    break;
+                                                default:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty 'default' clause.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)